cmd/editaccount.go

```diff
@@ -236,6 +243,21 @@ func (p *EditAccountParams) getTierLimits(tier int) (*jwt.JetStreamLimits, error
 	return src, nil
 }
 
+func (p *EditAccountParams) doDisableJetStream(r *store.Report) error {
+	if !p.disableJetStream {
+		return nil
+	}
+	p.claim.Limits.JetStreamLimits = jwt.JetStreamLimits{}
+	r.AddOK("deleted global limit")
+	if p.claim.Limits.JetStreamTieredLimits != nil {
```

Comment:
The nil check is not needed for iteration.

Reply:
fixed